velox/functions/sparksql/coverage/Coverage.cpp

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <gflags/gflags.h>
+#include <fstream>
+#include <iostream>
+#include "velox/exec/Aggregate.h"
+#include "velox/functions/CoverageUtil.h"
+#include "velox/functions/sparksql/Register.h"
+#include "velox/functions/sparksql/aggregates/Register.h"
+
+DEFINE_bool(all, false, "Generate coverage map for all Spark functions");
+
+using namespace facebook::velox;
+
+int main(int argc, char** argv) {
+  gflags::ParseCommandLineFlags(&argc, &argv, true);
+
+  // Register all simple and vector scalar functions.
+  functions::sparksql::registerFunctions("");
+
+  // Register Spark aggregate functions.
+  functions::sparksql::aggregates::registerAggregateFunctions("");
+
+  if (FLAGS_all) {
+    printCoverageMapForAll(":spark");
+  } else {
+    static const std::unordered_set<std::string> linkBlockList;
```

Comment:
This variable is not needed. Just pass "".

Reply:
For some reason, I see "in" without a link in the docs. Would you re-generate?

Reply:
Also, links do not work for me for some reason. Do these work for you?

Reply:
> This variable is not needed. Just pass "".
It seems that "" can not pass compile.
../../velox/functions/sparksql/coverage/Coverage.cpp:41:25: error: invalid initialization of reference of type ‘const std::unordered_set<std::__cxx11::basic_string<char> >&’ from expression of type ‘const char [1]’
   41 |     printVeloxFunctions("", ":spark");

Reply:
> Also, links do not work for me for some reason. Do these work for you?

I Also feel curious. I think some change in this pr cause link fail. because I found  link can work with velox latest main branch.

Reply:
> It seems that "" can not pass compile.

Oh. Sure, pass {}.

```
printVeloxFunctions({}, ":spark");
```

> I Also feel curious. I think some changes in this pr cause link fail. because I found link can work with velox latest main branch.

Would you investigate?